packages/typespec-azure-vscode/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "typespec-azure-vscode",
+  "displayName": "typespec azure",
+  "description": "the extension for azure typespec",
+  "publisher": "typespec",
+  "version": "0.0.1",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/chunyu3/typespec-azure-vscode.git"
+  },
+  "engines": {
+    "vscode": "^1.100.0"
+  },
+  "extensionDependencies": [
+    "typespec.typespec-vscode"
+  ],
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "onStartupFinished"
+  ],
+  "main": "./dist/src/extension.cjs",
+  "contributes": {
+    "commands": [
+      {
+        "command": "typespec.azure.emitCode",
+        "title": "Emit from TypeSpec Azure",
+        "category": "TypeSpec-azure"
+      }
+    ],
+    "languages": [
```

Comment:
Any reason to include this part? TypeSpec extension should have handled all these, isn't it?

Reply:
Even typespec-extension has defined the `typespec` language, but only when the typespec-extension has be activated, the `typespec` language can be see.
We cannot control the activate order, so we cannot make sure that the typespec-extension is activated before typespec-azure-extension.  When we open a tsp file at first, typespec-extension is activated, but typespec-azure-extension will not.
And we cannot make sure that typespec-extension will be installed at that time. (even we will install typespec-extension when we install typespec-azure-extension, typespec-extension may be uninstalled after that).

In order to make sure that typespec-extension can be triggered, we may need to duplicate this language here.

Reply:
OK. if it is acceptable not to consider typespec-extension uninstalled scenario, then we can remove the typespec language definition.